### ISSUE TITLE
go.dev: consider making header non-sticky 

### ISSUE THREAD

**Issue by dmage** [golang/go#37110]
### What is the URL of the page with the issue?
 
 https://pkg.go.dev/k8s.io/klog?tab=doc or any other package.
 
### What is your user agent?
 
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36
 
### Screenshot

> <img width="641" alt="Screenshot 2020-02-07 at 16 49 19" src="https://user-images.githubusercontent.com/443137/74043532-d8a27100-49c9-11ea-9a50-deff7f1162c9.png"> 
 
### What did you expect to see?
 
I'd expect my screen space to be used efficiently.
 
### What did you see instead?

A quite big header.

**Comment by arp242** [comment]
I've been using this userstyle to remove it, FYI:

    .Site-header { position: static; }  /* Unfix banner */
    :target::before { display: none; }  /* Undo fixing of scroll position with fixed header */

**Comment by julieqiu** [comment]
Thanks for the feedback!

We're currently discussing this request with our UX team, along with other design feedback we've received for the site. We want to review design feedback we have received as part of a broader redesign, as opposed to make piecemeal decisions. I'll update this issue once we have decided on any changes. 

/cc @fflewddur

**Comment by tooolbox** [comment]
If anything, this is probably the most important nav you would want to keep around:

![pkg go dev subnav](https://user-images.githubusercontent.com/4984708/76918807-66ecfb00-6884-11ea-84ea-f2365b08c45a.png)

**Comment by mholt** [comment]
I'm in favor of un-stickying it.

Specifically, the _least likely_ thing I am to do after scrolling deep into a package's docs is to look up another package or access the menu. I also agree that _if_ something is stickied on that page, it should be the tabs, and a very thin (`~1.5em` height) sticky at that.

**Comment by narqo** [comment]
Following [the tweet](https://twitter.com/tvii/status/1225100402582020096?s=20) I wrote last winter, I wanted to 👍 to the issue.

I have a strong belief, sticky headers shouldn't be used on the read-heavy sites.

As it's now pkg.go.dev team has added one extra bar to the site's header (i.e. "Support the Equal Justice Initiative"). I'm ok with the bar being there, but I wanted to point out, that in the way how it's implemented, it hurts the UX of the site. 

<img width="1552" alt="Screenshot 2020-06-17 at 09 14 54" src="https://user-images.githubusercontent.com/88045/84868825-8bbc5e80-b07d-11ea-9087-291ba2d345c1.png">

(_the comparison of heights, "useful" content vs headers, on 13" MBP, I use daily for work_)

**Comment by arp242** [comment]
Additional point which hasn't been raised: my laptop's screen is 1366x768, the usable screen size (without Chrome's UI chrome) is 675px, and the banner is 105px. This means that the banner takes up 15% of my screen size with the default zoom level (more if I zoom in!), which is not an insignificant amount on an already small screen.

The issue is less pronounced with a larger resolution, but I think it would be a good thing to ensure the Go documentation is as accessible as we can make it, including for people of limited financial means (possibly from a disadvantages background and/or parts of the world where most of the population are of limited financial means) who use comparatively "outdated" hardware.

In other words, I think it's a little bit more than "just an UX issue". Ironically, putting the BLM banner there actually made the site harder to use especially for people from a lower economical class (which are disproportionally black in the US) wanting to learn/use Go, which I'm pretty sure wasn't the intention with that.

**Comment by arp242** [comment]
Note the new banner sits over the scrollbar now: 

![screenshot_2020-07-31-04-44-49_border](https://user-images.githubusercontent.com/1032692/88972758-a0c91600-d2e8-11ea-8a62-c65845f5091a.png)

Also, after clicking on an item in the header it seems the focus is "stuck" somewhere and can no longer use page up, home, arrow down, or other navigation keys. This happens in both Firefox and Chromium.

It also doesn't address the issue really: it's still a fixed banner taking up screen space better used for actually reading the documentation. Arguably it's even worse because it's now a distracting animated thing (never mind the bugs). I don't get what's the problem with a simple static web page with text 🤷‍♂️

At any rate, here's a new userstyle to disable all of this again, for whom it may concern:

```
.DetailsNavFixed { display: none; }   /* Unfix banner */
:target::before { display: none;; }   /* Undo hack to fix scroll position with fixed header */
```

**Comment by tooolbox** [comment]
I do think it's a more useful menu than the original (which is what I said in my comment).

However, from playing with it and using it, I think that having the screen real estate for viewing the docs is more valuable to me than having the navigation always available.

As much as it must be annoying to thrash around on this single point, could we try no sticky banner at all and see how that goes?  No matter the answer, I really appreciate the Go Team's attention on this seemingly simple matter :)

**Comment by shabbyrobe** [comment]
Should this be closed? There's still a big sticky header, now with a distracting animation. I too would prefer the screen area be left clear for reading and reading alone. None of the information stuck in the header is useful to me below the fold while I'm trying to read through package documentation.
